kvx/meson.build

```diff
@@ -0,0 +1,20 @@
+project('kvx', 'c')
+
+rz_asm_dep = dependency('rz_asm')
+rz_analysis_dep = dependency('rz_asm')
+plugins_dir = get_option('prefix') / rz_asm_dep.get_variable(pkgconfig: 'plugindir', cmake: 'rz_asm_PLUGINDIR')
```

Comment:
change this to
```py
rizin_plugdir = get_option('rizin_plugdir')
if rizin_plugdir == ''
  plugins_dir = rz_core_dep.get_variable(pkgconfig: 'plugindir', cmake: 'rz_asm_PLUGINDIR')
endif
```

and create a new file like this one https://github.com/rizinorg/rz-libyara/blob/main/meson_options.txt containing just the `rizin_plugdir` line

Reply:
ok